tee-worker/ts-tests/integration-tests/common/utils/vc-helper.ts

```diff
@@ -166,85 +166,3 @@ export const unconfiguredAssertions = [
         },
     },
 ];
-export const jsonSchema = {
```

Comment:
I tested it locally, and it works. In fact, this crude jsonSchema should have been removed and replaced long ago. It was created during the first VC request. Thanks for your repo!

Reply:
Thanks @Verin1005 , if there is anything against, can I get the approval check to merge it? 😄